packages/dm-core-plugins/src/responsive_grid/ResponsiveGridPlugin.tsx

```diff
@@ -0,0 +1,54 @@
+import { IUIPlugin, Stack, ViewCreator } from '@development-framework/dm-core'
+import { Typography } from '@equinor/eds-core-react'
+import React from 'react'
+import { Col, Container, Row } from 'react-grid-system'
+import { TCol, TGridPluginConfig, TRow } from './types'
+
+const defaultConfig: TGridPluginConfig = {
+  rows: [],
+}
+
+export const ResponsiveGridPlugin = (
+  props: IUIPlugin & { config: TGridPluginConfig }
+): React.ReactElement => {
+  const { config, idReference, type, onSubmit, onChange } = props
+
+  const internalConfig: TGridPluginConfig = {
+    ...defaultConfig,
+    ...config,
+  }
+
+  return (
+    <Stack
+      style={{
+        width: '100%',
+        height: '100%',
+        display: 'flex',
+        maxWidth: 'max-content',
+      }}
+    >
+      <Container>
+        {internalConfig.rows.map((row: TRow) => {
+          const columns = row.columns.map((col: TCol) => {
+            return (
+              <Col {...col.size}>
+                <div style={{ marginBottom: '40px' }}>
+                  {col?.title && (
+                    <Typography variant='h4'>{col.title}</Typography>
+                  )}
+                  <ViewCreator
+                    idReference={idReference}
+                    viewConfig={col.viewConfig}
+                    onSubmit={onSubmit}
+                    onChange={onChange}
+                  />
+                </div>
+              </Col>
+            )
+          })
+          return <Row gutterWidth={row.gutterWidth || 60}>{columns}</Row>
```

Comment:
Also think default should be smaller here. I assume gutterWidth is the same as gap.
 I think rowGap and columnGap makes sense to use and to define it as such in the plugin blueprint. Maybe there are use-cases where columnGap is different per row, but I don't think it's very common. 

Or keep gutterWidth and set default to 0, but allow user to set gutterWidth if they want different values per row + add use of column-gap and row-gap which is the default